src/game/mario_actions_airborne.c

```diff
@@ -539,6 +539,7 @@ s32 act_freefall(struct MarioState *m) {
     }
 
     switch (m->actionArg) {
+        default:
```

Comment:
i think this is a little hacky looking, having a default fallthrough above a case just makes me think that either
- case ACT_ARG_FREEFALL_GENERAL could be removed
- just initialize `animation` to `MARIO_ANIM_GENERAL_FALL` and remove the default case